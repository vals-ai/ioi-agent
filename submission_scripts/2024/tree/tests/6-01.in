version https://git-lfs.github.com/spec/v1
oid sha256:154914760d8fdb965b99bf4ee8df98fdad05249a02d6afd321e010efa755bdd4
size 2698735
