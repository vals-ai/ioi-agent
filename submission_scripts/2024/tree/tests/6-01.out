version https://git-lfs.github.com/spec/v1
oid sha256:6b538cb889191ad1ca3896c0d49d26dff05166c8c1e9e238f29b68adced1ca7b
size 48048
